test/test.js

```diff
@@ -0,0 +1,47 @@
+const mongoose = require('mongoose');
+const db = require('../database/index');
+
+let chai = require('chai');
+let chaiHttp = require('chai-http');
+let app = require('../server/index');
+
+let should = chai.should();
+chai.use(chaiHttp);
+
+describe('/GET hashtags', () => {
+  it('it should GET all the hashtags', (done) => {
```

Comment:
For a few items (< 1000) it might seem ok to write a "Get ALL the things" route. Don't trust that it'll be depracated when there are more items. Add a default limit to the query to the database and as a stretch, add a query param to modify the limit.

Same for users on the other service.